test/TestAsset/Parameters.php

```diff
@@ -23,7 +23,7 @@ public function typelessRequired($bar)
     {
     }
 
-    public function typehintOptional(A $fooOpt = null)
+    public function typehintOptional(?A $fooOpt = null)
```

Comment:
Is it not affecting tests? Usually we don't want to run cs checks on TestAssets

Reply:
No it shouldn't since this just explicitly states the already implied nullabillity of this param. Should tests in general be excluded or just the test assets?

Reply:
TestAssets please

```xml
<exclude-pattern>*/TestAsset/*</exclude-pattern>
```